User Journey Encarni 👽

carlotiii30 opened the pull request.

marinajcs committed:
Creado pull_request_template.md

marinajcs committed:
Corregido markdown lint pull_request_template.md

carlotiii30 committed:
User Journey Encarni 👽

marinajcs committed:
Modificado pull_request_template.md

Respetado límite de 80 caracteres por línea

carlotiii30 committed:
Update user_journey-encarni.md

pluque01 commented on user_journey-encarni.md:
```diff
@@ -0,0 +1,9 @@
+## User Journey Encarni
```
El título principal debe empezar con un #

pluque01 commented on user_journey-encarni.md:
```diff
@@ -0,0 +1,9 @@
+## User Journey Encarni
+
+Mi nombre es Encarni, soy jefa de proyectos en una importante empresa de Granada, todo un equipo depende de mi. Dos veces al año, en mayo y en septiembre, nos reúnen a todos los integrantes del equipo para decidir la organización de nuestras vacaciones. 
```
El linter avisa de que sobra un espacio al final, pero también va a dar fallo si no pones un límite de 80 caracteres por línea

pluque01 commented on user_journey-encarni.md:
```diff
@@ -0,0 +1,9 @@
+## User Journey Encarni
+
+Mi nombre es Encarni, soy jefa de proyectos en una importante empresa de Granada, todo un equipo depende de mi. Dos veces al año, en mayo y en septiembre, nos reúnen a todos los integrantes del equipo para decidir la organización de nuestras vacaciones. 
+Teniendo en cuenta que, sin un jefe de proyecto, el trabajo no puede avanzar, el equipo tiene que ponerse de acuerdo en la elección del periodo vacacional. Además, hay que cumplir ciertas condiciones:
```
Aquí va a fallar por lo mismo, el límite de 80 caracteres por línea.

carlotiii30 committed:
Merge remote-tracking branch 'origin/main' into carlotiii30-patch-1

carlotiii30 committed:
Correcciones user journey

carlotiii30 committed:
Update user_journey-encarni.md

carlotiii30 committed:
Update user_journey-encarni.md

marinajcs committed:
Última modificación pull_request_template.md

Eliminada mayúscula arbitraria

carlotiii30 committed:
Update user_journey-encarni.md

puchy22 reviewed (changes requested):
Creo que sería conveniente añadir en más detalle qué es lo que hace Encarni desde que abre la aplicación hasta que la cierra.

moshidev committed:
:busts_in_silhouette::briefcase: user stories doc - Recursos Humanos (#6)

* añade borrador user journey hr

* Update user-journey-hr.md

* :heavy_plus_sign: :rotating_light: #1

* Añadida descripción del problema

* :see_no_evil:

* :recycle: para testear el linter

* Corrige :poop:

* :recycle: para testear el linter

* :recycle:

* Update user-journey-hr.md

* Update user-journey-hr.md

* Update user-journey-hr.md

* Update user-journey-hr.md

* corrige directorio y estilo markdown

* corrige ortografía

* :skull:

* :flag_spain:

* elimina workflow supeditado por mkcheck.md

* mejora historia de usuario hr

---------

Co-authored-by: Carlos Servillera Hernandez <83367971+carlosservi@users.noreply.github.com>
Co-authored-by: JJ Merelo <jjmerelo@gmail.com>
Co-authored-by: Pablo Luque Salguero <pablols114@gmail.com>

Enaraque committed:
Corregido user journey currante (#25)

* Actualizando readme y nombre de fichero

* Añadido fichero

danieeeld2 committed:
📝Documentación sobre como gestionan las empresas las vacaciones📝 (#3)

* Creando Archivo

* Update InformacionExtra.md

* Corregidas faltas de ortografía

* Actualización sobre las bajas y ortografía

Se ha completado la información de los trabajadores que tienen una baja y arreglado la ortografía

* Update InformacionExtra.md - días hábiles

* Update-InformacionExtra.md-"\n"

---------

Co-authored-by: Raul <93154854+raultl12@users.noreply.github.com>
Co-authored-by: Mihnea Vidican <66739635+johnwaves@users.noreply.github.com>
Co-authored-by: javigp2002 <72984347+javigp2002@users.noreply.github.com>
Co-authored-by: Juan Fernández de Cañete Verdejo <77067504+DarckMonster@users.noreply.github.com>

Enaraque commented:
Ha fallado el workflow por el md de currante. Ya ha sido arreglado en el issue #24. 
Actualizad la rama para que no falle y asi se pueda hacer el merge 🤝

JJ commented:
> Ha fallado el workflow por el md de currante. Ya ha sido arreglado en el issue #24. Actualizad la rama para que no falle y asi se pueda hacer el merge 🤝

Lo puede hacer cualquiera con permiso de escritura :+1:

shvtwp committed:
🧍‍♂️Añadida persona currante (Miguel) (#22)

* 🧍‍♂️Añadida persona currante (Miguel)

* Solución de faltas en persona-currante.md

* Solución de faltas persona-currante.md

* Errores persona-currante.md

Paco miguel

* Update persona-currante.md

* Solucionados errores de md persona-currante.md

JJ committed:
[🐛 ] Arregla el comprobador ortográfico (#30)

* :bug: arregla paths closes #16

Falla creo que por el cambio de paths

* :memo: para lanzar tests

* :rotating_light:

* Hay que poner todos los ficheros :poop:

* :memo: para lanzar tests

* :rotating_light:

javigp2002 commented on user_journey-encarni.md:
```diff
@@ -0,0 +1,19 @@
+vacaciones, puede transferirse a otro equipo para que continúe trabajando con
+él.
+
+Cada equipo solicitará, en la aplicación, su periodo de preferencia, que podrá
```
¿Lo solicitá entonces para una persona o se pide para un equipo completo?

javigp2002 reviewed (commented):
Está bien el user journey aunque quizás sea más HU que user journey a mi parecer. No obstante, miraría el problema que comento en el código.

carlotiii30 committed:
Update user_journey-encarni.md

javigp2002 reviewed (approved):
:+1:

carlotiii30 committed:
Cambios en user journey

puchy22 commented:
:+1:
